pheweb/serve/static/custom_locuszooms.js

```diff
@@ -196,11 +196,14 @@ LocusZoom.Data.FG_LDDataSource.prototype.getURL = function(state, chain, fields)
     chain.header.ldrefvar = topvar
     if (window.lz_conf.ld_service.toLowerCase() == 'finngen') {
 	var windowSize = Math.min(state.end - state.start + 10000, window.lz_conf.ld_max_window)
-	return refvar ? this.url + "?variant=" + topvar['association:chr'] + ':' + topvar['association:position'] + ':' + topvar['association:ref'] + ':' + topvar['association:alt'] + "&window=" + windowSize + "&panel=sisu3" : this.url + 'couldntgetrefvar'
+	return this.url + "?variant=" + topvar['association:chr'] + ':' + topvar['association:position'] + ':' + topvar['association:ref'] + ':' + topvar['association:alt'] + "&window=" + windowSize + "&panel=sisu3"
     } else {
 	var windowSize = 500
-	var population="1000GENOMES:phase_3:FIN"
-	return refvar ? this.url + refvar + "/" + population + "?window_size=" + windowSize : this.url + 'couldntgetrefvar'
+	var population="1000GENOMES:phase_3:ALL"
```

Comment:
could this be default conf and the population could be configured to something else?